test/e2e/middleware-general/test/index.test.ts

```diff
@@ -102,6 +114,34 @@ describe('Middleware Runtime', () => {
   }
 
   function runTests({ i18n }: { i18n?: boolean }) {
+    if (isNodeMiddleware) {
+      it('should be able to use node builtins with node runtime', async () => {
+        const res = await next.fetch('/test-node-fs')
+        expect(res.status).toBe(200)
+
+        const body = await res.json()
+        expect(body.dependencies || body.devDepependencies).toBeTruthy()
```

Comment:
typo or by design? is test not catching this?

```suggestion
        expect(body.dependencies || body.devDependencies).toBeTruthy()
```

Reply:
test doesn't catch as it hits first condition, second is just incase we switch